lib/components/global/modal/ModalContainer.js

```diff
@@ -272,8 +272,22 @@ const ModalContainer = React.createClass({
 	 * @return {object} Width of modal in pixels
 	 */
 	_modalWidth: function() {
+		let width = this.props.width;
+		try {
+			if (! this.unitRegExp.test(width)) {
+				let newWidth = width.match(this.digitRegExp)[0] + 'px';
+
+				console.warn(`The value provided for width: "${width}" was not valid.  Adjusting to ${newWidth}. Please note that valid css units are required: https://developer.mozilla.org/en-US/docs/Web/CSS/length`);
+				width = newWidth;
+			}
+
+		} catch (e) {
+			console.warn(`A digit is required when specifying width.  Your input: "${width}".  Falling back to 600px.`);
+			width = "600px";
```

Comment:
This default could live in the CSS like the Tooltip does.  That way, we don't need to set a default in the JS.